config-paths.js

```diff
@@ -0,0 +1,23 @@
+const { app } = require('electron');
+const { homedir } = require('os');
+const { resolve } = require('path');
+const { existsSync } = require('fs');
+
+const home = homedir();
+const root = resolve(app.getPath('appData'), 'HyperTerm');
```

Comment:
@Okahyphen That app name is always `Electron` during dev as it's the packager that changes the app name in Info.plist to what's defined in `productName` in package.json. Just look at the name in the top left of the menubar when running it with `npm test`, it says Electron, not HyperTerm. I would personally consider that a feature, as you don't mess up your own config when developing on HyperTerm.